interfaces.go

```diff
@@ -1,30 +1,7 @@
 package pack
```

Comment:
The remnants here look out of place.

Reply:
See response in `client.go`.

Reply:
Agreed it's sort of weird to have them here, and I'm happy to disperse them to their homes.

Reply:
I think these were grouped here for convenience, but now that we have `make generate`, there's no need.